src/include/migraphx/matcher.hpp

```diff
@@ -878,7 +878,7 @@ inline auto literal_value_checker(F f)
  * Uses no tolerance for integral types.
  */
 template <class T>
-inline auto has_value(T x, std::size_t atol_mult = 10, std::size_t rtol_mult = 10)
+inline auto has_value(T x, float atol_mult = 10, float rtol_mult = 10)
```

Comment:
I think its better to have that as a integer to make it clearer its a multiple of epsilon and not epsilon. You can set `atol_mul` to 0 to handle small numbers.

Reply:
I  was interpreting `eps` as scaling factor for different dtypes.  What you are suggesting is interpreting atol and rtol as multiples/scaling factors.  I don't have problem with either.